test/core/flows/flow-time-updater/flow-time-updater-function.test.ts

```diff
@@ -0,0 +1,45 @@
+import { Duration } from 'aws-cdk-lib';
+import { Schedule } from 'aws-cdk-lib/aws-events';
+import type { CloudFormationCustomResourceEvent } from 'aws-lambda';
+import * as API from '../../../../src/core/flows/flow-time-updater/api';
+import { handler } from '../../../../src/core/flows/flow-time-updater/flow-time-updater.lambda';
+
+describe('flow time updater function', () => {
+  it('converts CDK rate expressions to AppFlow rate expressions', async () => {
```

Comment:
I added a test for the new behavior. I didn't have the bandwidth to backfill other tests, but this test file could be updated in the future to improve test coverage.